app/uk/gov/hmrc/nisp/controllers/AccountController.scala

```diff
@@ -59,9 +58,9 @@ trait AccountController extends NispFrontendController with AuthorisedForNisp wi
       val authenticationProvider = getAuthenticationProvider(user.authContext.user.confidenceLevel)
 
       nispConnector.connectToGetSPResponse(nino).map {
-        case SPResponseModel(Some(spSummary: SPSummaryModel), None, None) => getABTest(nino, spSummary.contractedOutFlag)
+        case SPResponseModel(Some(spSummary: SPSummaryModel), None, None) => spSummary.contractedOutFlag
           match {
-            case Some(ABTest.B) => Ok(account_cope(nino, spSummary.forecast.forecastAmount.week,
+            case true => Ok(account_cope(nino, spSummary.forecast.forecastAmount.week,
```

Comment:
This could probably just be an if statement instead as you are not extracting any values in the pattern matching